Makefile

```diff
@@ -100,7 +100,7 @@ RESTIC_VERSION ?= 0.15.0
 
 CLI_PLATFORMS ?= linux-amd64 linux-arm linux-arm64 darwin-amd64 darwin-arm64 windows-amd64 linux-ppc64le
 BUILDX_PLATFORMS ?= $(subst -,/,$(ARCH))
-BUILDX_OUTPUT_TYPE ?= docker
+BUILDX_OUTPUT_TYPE ?= image
```

Comment:
https://docs.docker.com/build/exporters/image-registry/
According to the document, we should add parameters to let the image be pushed to destinations automatically.
For example `BUILDX_OUTPUT_TYPE ?= image,push=true`.

The original `BUILDX_OUTPUT_TYPE ?= docker` will push the images after build.

Reply:
> The original BUILDX_OUTPUT_TYPE ?= docker will push the images after build.

I didn't find that [to be the case in the docs](https://docs.docker.com/build/exporters/#:~:text=Exporters%20save%20your,creates%20a%20cache.).. The only thing that would push automatically is 

```
--output=type=registry
```

> Exporters save your build results to a specified output type. You specify the exporter to use with the [--output CLI option](https://docs.docker.com/reference/cli/docker/buildx/build/#output). Buildx supports the following exporters:
> 
> image: exports the build result to a container image.
> registry: exports the build result into a container image, and pushes it to the specified registry.
> local: exports the build root filesystem into a local directory.
> tar: packs the build root filesystem into a local tarball.
> oci: exports the build result to the local filesystem in the [OCI image layout](https://github.com/opencontainers/image-spec/blob/v1.0.1/image-layout.md) format.
> docker: exports the build result to the local filesystem in the [Docker Image Specification v1.2.0](https://github.com/moby/moby/blob/v25.0.0/image/spec/v1.2.md) format.
> cacheonly: doesn't export a build output, but runs the build and creates a cache.

Reply:
> should add parameters to let the image be pushed to destinations automatically.
For example BUILDX_OUTPUT_TYPE ?= image,push=true

will do.

Reply:
done via https://github.com/vmware-tanzu/velero/pull/8252/commits/d97885b6c3c3d93a3db4cec09a035f00cdd49129